### PR TITLE
User/andnguyen/doc

### PR DESCRIPTION
**Pull request by secretbetta**
Added documentation for easy understanding. And it also shows how to make C# documentation

### CODE REVIEW

File: BassBotDotNET/Program.cs [hunk 1]
```diff
@@ -4,14 +4,23 @@
 using Discord.WebSocket;
 using Newtonsoft.Json;
 
+namespace BassBotDotNET;
+
+/// <summary>
+/// Main program to run the discord bot.
+/// </summary>
```
Comment (jghuynh) [inline BassBotDotNET/Program.cs:11]: Oh do you comment by using 3 positive slashes? And some of the comments remind me of html.

Comment (secretbetta) [reply BassBotDotNET/Program.cs:11]: two slashes can make a comment. but the formatting for method/class documentation looks like this ^

File: BassBotDotNET/Program.cs [hunk 2]
```diff
@@ -4,14 +4,23 @@
...
+    public static Task Main() => new Program().MainAsync();
 
     private DiscordSocketClient? _client;
 
+    // Settings command module
     private readonly SettingSlashCommandModule _settingSlashCommandModule = new();
 
+    /// <summary>
+    /// Main method for the bot.
+    /// </summary>
```
Comment (jghuynh) [inline BassBotDotNET/Program.cs:23]: Ohhh is this similar to the Constructor in Java?

File: BassBotDotNET/Program.cs [hunk 3]
```diff
@@ -32,12 +41,21 @@ public async Task MainAsync()
         await Task.Delay(-1);
     }
 
+    /// <summary>
+    /// Basic message logger.
+    /// </summary>
+    /// <param name="msg"></param>
+    /// <returns></returns>
```
Comment (jghuynh) [inline BassBotDotNET/Program.cs]: I noticed that you don't write the type of return variable in the C# doc.

**Review by jghuynh — APPROVED** [review]
These C# documentation looks amazing! Thank you!

File: BassBotDotNET/Program.cs (hunk 2 above)
Comment (jghuynh) [reply BassBotDotNET/Program.cs:23]: Nope that is not a Constructor in C#

File: BassBotDotNET/Program.cs (hunk 3 above)
Comment (secretbetta) [reply BassBotDotNET/Program.cs]: Thanks for catcching this, this is a mistake LOL

File: BassBotDotNET/Program.cs (hunk 2 above)
Comment (secretbetta) [reply BassBotDotNET/Program.cs:23]: xD This is more of a the .NET specific library for discord. I'm not entirely sure how it works either though.

File: BassBotDotNET/Program.cs (hunk 3 above)
Comment (secretbetta) [reply BassBotDotNET/Program.cs]: I apparently missed the <param> comment as well omfg. I think I was rushing while making the docs LOL

Comment (secretbetta) [reply BassBotDotNET/Program.cs]: AS WELL AS ALOT OF OTHER PARAM AND RETURN DOCS my god

**Review by jghuynh — APPROVED** [review]
Looks amazing!

**Pull request merged**